docs/en/concepts-and-designs/kubesphere-monitoring.md

```diff
@@ -0,0 +1,43 @@
+# KubeSphere Monitoring
+
+## Overview
+
+The KubeSphere monitoring system comprises many components that work together to achieve overall monitoring functionality. KubeSphere defaults to installing the monitoring module. It will deploy  or create a few components within the namespace `kubesphere-monitoring-system`:
+
+- Prometheus Operator: automates tasks for operating Prometheus instances.
+- Prometheus Servers: scrapes metrics and provides monitoring service.
+- Kube-state-metrics: exposes kubernetes resource metrics.
+- Node-exporter: exposes node metrics.
+
+Besides, the monitoring stack includes CRD resources:
+
+- ServiceMonitor: defines monitoring for a set of services.
+- Prometheus: defines Prometheus deployments, `k8s` and `k8s-system`.
+- PrometheusRule: describes recording rules.
+
```

Comment:
Maybe we should not mention details like setup details like k8s/ k8s-system / prometheus-k8s / prometheus-k8s-system ? Because these will be changed in 3.0.